javers-core/src/test/groovy/org/javers/core/json/typeadapter/CdoSnapshotTypeAdapterTest.groovy

```diff
@@ -41,6 +40,7 @@ class CdoSnapshotTypeAdapterTest extends Specification {
         json.globalId.entity == "org.javers.core.model.DummyUser"
         json.globalId.cdoId == "kaz"
         json.type == "INITIAL"
+        json.version == 1
```

Comment:
I suggest to test it in a different (minimal) way.
So test new version field only in 2 basic tests:
`def "should serialize CdoSnapshot to Json"()` and
`def "should deserialize CdoSnapshot"()`
and not to change other tests.
Deserializer should not break if field is missing (and it will be missing if users ugrade to new JaVers version and will load snapshots created in old JaVers version).